mediacenter/models.py

```diff
@@ -0,0 +1,78 @@
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ideasbox.models import TimeStampedModel
+from search.models import SearchableQuerySet, SearchMixin
+from .utils import guess_type
+
+
+class DocumentQuerySet(SearchableQuerySet, models.QuerySet):
+    def image(self):
+        return self.filter(type=Document.IMAGE)
+
+    def video(self):
+        return self.filter(type=Document.VIDEO)
+
+    def pdf(self):
+        return self.filter(type=Document.PDF)
+
+    def text(self):
+        return self.filter(type=Document.TEXT)
+
+    def audio(self):
+        return self.filter(type=Document.AUDIO)
+
+
+class Document(SearchMixin, TimeStampedModel, models.Model):
```

Comment:
(I know it's not related to this PR, but there is some inconsistency in the names, here: `TimeStampedModel` is a mixin or a model.Model subclass ?)